scripts/install

```diff
@@ -1,117 +1,94 @@
+ynh_app_setting_set --app=$app --key=admin --value=$admin
+ynh_app_setting_set --app=$app --key=is_public --value=$is_public
+ynh_app_setting_set --app=$app --key=deskey --value=$deskey
 
 #=================================================
 # STANDARD MODIFICATIONS
 #=================================================
 # INSTALL DEPENDENCIES
 #=================================================
-ynh_script_progression --message="Installing dependencies..." --time --weight=1
+ynh_script_progression --message="Installing dependencies..." --weight=1
 
 ynh_install_app_dependencies $pkg_dependencies
 
 #=================================================
-# INSTALL PHP
-#=================================================
-ynh_script_progression --message="Installing PHP $php_app_version..." --time --weight=3
-
-ynh_install_php --phpversion="$php_app_version" --package="$php_additonal_packages"
-
-#=================================================
-# CREATE A MYSQL DB
+# CREATE A MYSQL DATABASE
 #=================================================
-ynh_script_progression --message="Creating a MySQL database..." --time --weight=1
+ynh_script_progression --message="Creating a MySQL database..." --weight=1
 
-db_name=$(ynh_sanitize_dbid $app)
+db_name=$(ynh_sanitize_dbid --db_name=$app)
 db_user="$db_name"
-
-# Generate random DES key & password
-deskey=$(ynh_string_random 24)
-ynh_app_setting_set "$app" deskey "$deskey"
-ynh_app_setting_set "$app" db_name "$db_name"
-
-# Initialize database
-ynh_mysql_setup_db "$db_user" "$db_name"
+ynh_app_setting_set --app=$app --key=db_name --value=$db_name
+ynh_mysql_setup_db --db_user=$db_user --db_name=$db_name
 
 #=================================================
 # DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
-ynh_script_progression --message="Downloading source files..." --time --weight=1
+ynh_script_progression --message="Setting up source files..." --weight=1
 
-ynh_app_setting_set $app final_path "$final_path"
-# Create tmp directory and fetch app inside
-TMPDIR=$(mktemp -d)
-ynh_setup_source "$TMPDIR"
-
-#=================================================
-# CREATE DEDICATED USER
-#=================================================
-ynh_script_progression --message="Configuring system user..." --time --weight=1
-
-ynh_system_user_create $app # Create a dedicated system user
+ynh_app_setting_set --app=$app --key=final_path --value=$final_path
+# Download, check integrity, uncompress and patch the source from app.src
+ynh_setup_source --dest_dir="$final_path"
 
 #=================================================
 # NGINX CONFIGURATION
 #=================================================
-ynh_script_progression --message="Configuring nginx web server..." --time --weight=1
+ynh_script_progression --message="Configuring nginx web server..." --weight=1
 
+# Create a dedicated nginx config
 ynh_add_nginx_config
+
 if [ "$path_url" = "/" ]
```

Comment:
This part is not in the example package, is it still necessary ? What do you think @yalh76 ?

Reply:
I'm not sure it's necessary but haven't the time to test without, so I keep it